src/fusion_power_plant.cc

```diff
@@ -12,45 +12,71 @@ std::string FusionPowerPlant::str() {
 
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::EnterNotify() {
-  //Pseudocode implementation of EnterNotify()
   cyclus::Facility::EnterNotify();
-  fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / seconds_per_year * context()->dt());
-  fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
-  blanket_turnover = blanket_size * blanket_turnover_rate;
 
-  //Create the blanket material for use in the core
-  const cyclus::Composition::Ptr enriched_li = context->GetRecipe(blanket_inrecipe);
-  cyclus::Material::Ptr blanket = cyclus::Composition::CreateFromAtom(0.0, enriched_li);
+  //fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / 
+  //  seconds_per_year * context()->dt());
+  //fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
+  //blanket_turnover = blanket_size * blanket_turnover_rate;
 
-  //Truncated for readability. It goes on Tritium Storage and gets started, though.
-  fuel_startup_policy.Init(&tritium_storage).Set(fuel_incommod).Start();
-  
-  //Truncated for readability
-  blanket_fill_policy.Init(&blanket).Set(blanket_incommod).Start();
+  //Create the blanket material for use in the core, no idea if this works...
+  const Composition::Ptr enriched_li = context->GetRecipe(blanket_inrecipe);
+  blanket = Composition::CreateFromAtom(0.0, enriched_li);
+
+  fuel_startup_policy
+      .Init(this, &tritium_storage, std::string("Tritium Storage"),
+            &fuel_tracker)
+      .Set(fuel_incommod)
+      .Start();
 
-  //Tritium Buy Policy Section:
+  blanket_fill_policy
+      .Init(this, &blanket_feed, std::string("Blanket Startup"), 
+            &blanket_tracker)
+      .Set(blanket_incommod)
+      .Start();
+
+  // Tritium Buy Policy Selection:
   if (refuel_mode == "schedule") {
-    IntDistribution::Ptr active_dist =
-        FixedIntDist::Ptr(new FixedIntDist(1));
-    IntDistribution::Ptr dormant_dist =
-        FixedIntDist::Ptr(new FixedIntDist(buy_frequency - 1));
-    DoubleDistribution::Ptr size_dist =
-        FixedDoubleDist::Ptr(new FixedDoubleDist(1));
-    //Do not start the policy yet.
+    IntDistribution::Ptr active_dist = 
```

Comment:
You probably need to add some header to have access to these.

Reply:
I'll double check that and make the necessary changes